data/links.yaml

```diff
@@ -59,6 +59,27 @@ inputs:
   WhiteStripe:
     Data: ftp://www.nitrc.org/home/groups/captk/downloads/SampleData_1.6.0/WhiteStripe.zip
     Model: N.A.
+appsDownload:
```

Comment:
Why is there download information in 2 separate files? Remove application download information from one of them.